scripts/demo_bonds_augmented_alpha_1_sells_disabled.sh

```diff
@@ -0,0 +1,190 @@
+  "verifyKey":"Ftsqjc2pEvGLqBtgvVx69VXLe1dj2mFzoi4kqQNGo3Ej",
+  "encryptionPublicKey":"8YScf3mY4eeHoxDT9MRxiuGX5Fw7edWFnwHpgWYSn1si",
+  "secret":{
+    "seed":"94f3c48a9b19b4881e582ba80f5767cd3f3c5d7b7103cb9a50fa018f108d89de",
+    "signKey":"B2Svs8GoQnUJHg8W2Ch7J53Goq36AaF6C6W4PD2MCPrM",
+    "encryptionPrivateKey":"B2Svs8GoQnUJHg8W2Ch7J53Goq36AaF6C6W4PD2MCPrM"
+  }
+}'
+SHAUN_DID_FULL='{
+  "did":"did:ixo:U4tSpzzv91HHqWW1YmFkHJ",
+  "verifyKey":"FkeDue5it82taeheMprdaPrctfK3DeVV9NnEPYDgwwRG",
+  "encryptionPublicKey":"DtdGbZB2nSQvwhs6QoN5Cd8JTxWgfVRAGVKfxj8LA15i",
+  "secret":{
+    "seed":"6ef0002659d260a0bbad194d1aa28650ccea6c6862f994dfdbd48648e1a05c5e",
+    "signKey":"8U474VrG2QiUFKfeNnS84CAsqHdmVRjEx4vQje122ycR",
+    "encryptionPrivateKey":"8U474VrG2QiUFKfeNnS84CAsqHdmVRjEx4vQje122ycR"
+  }
+}'
+
+# Ledger DIDs
+echo "Ledgering DID 1/3..."
+ixocli tx did add-did-doc "$MIGUEL_DID_FULL" --broadcast-mode block --gas-prices="$GAS_PRICES" -y
+echo "Ledgering DID 2/3..."
+ixocli tx did add-did-doc "$FRANCESCO_DID_FULL" --broadcast-mode block --gas-prices="$GAS_PRICES" -y
+echo "Ledgering DID 3/3..."
+ixocli tx did add-did-doc "$SHAUN_DID_FULL" --broadcast-mode block --gas-prices="$GAS_PRICES" -y
+
+# d0 := 1000000 // initial raise (reserve)
+# p0 := 1       // initial price (reserve per token)
+# theta := 0    // initial allocation (percentage)
+# kappa := 3    // degrees of polynomial (i.e. x^2, x^4, x^6)
+
+# R0 = 1000000        // initial reserve (1-theta)*d0
+# S0 = 1000000        // initial supply
+# V0 = 1000000000000  // invariant
+
+echo "Creating bond..."
+ixocli tx bonds create-bond \
+  --token=abc \
+  --name="A B C" \
+  --description="Description about A B C" \
+  --function-type=augmented_function \
+  --function-parameters="d0:1000000,p0:1,theta:0,kappa:3.0" \
+  --reserve-tokens=res \
+  --tx-fee-percentage=0 \
+  --exit-fee-percentage=0 \
+  --fee-address="$FEE" \
+  --max-supply=20000000abc \
+  --order-quantity-limits="" \
+  --sanity-rate="0" \
+  --sanity-margin-percentage="0" \
+  --alpha-bond \
+  --batch-blocks=1 \
+  --outcome-payment="300000000" \
+  --bond-did="$BOND_DID" \
+  --creator-did="$MIGUEL_DID_FULL" \
+  --controller-did="$FRANCESCO_DID" \
+  --broadcast-mode block --gas-prices="$GAS_PRICES" -y
+echo "Created bond..."
+ixocli q bonds bond "$BOND_DID"
+
+echo "Miguel buys 400000abc..."
+ixocli tx bonds buy 400000abc 500000res "$BOND_DID" "$MIGUEL_DID_FULL" --broadcast-mode block --gas-prices="$GAS_PRICES" -y
+echo "Miguel's account..."
+ixocli q auth account "$MIGUEL_ADDR"
+
+echo "Francesco buys 400000abc..."
+ixocli tx bonds buy 400000abc 500000res "$BOND_DID" "$FRANCESCO_DID_FULL" --broadcast-mode block --gas-prices="$GAS_PRICES" -y
+echo "Francesco's account..."
+ixocli q auth account "$FRANCESCO_ADDR"
+
+echo "Shaun cannot buy 200001abc..."
+ixocli tx bonds buy 200001abc 500000res "$BOND_DID" "$SHAUN_DID_FULL" --broadcast-mode block --gas-prices="$GAS_PRICES" -y
+echo "Shaun cannot sell anything..."
```

Comment:
Independently of the allow sells changes, Shaun shouldn't be able to sell because he didn't buy, so I think we can remove Shaun selling here.

Reply:
Did not remove but commented it out and added a reason why in the comment, so that this script matches with the `*_sells_enabled` script for the most part.